src/components/EventCard/index.tsx

```diff
@@ -9,36 +11,39 @@ const EventCard: React.FC<{
   location: string;
   time: string;
   facebookUrl?: string;
-}> = ({ month, date, day, title, org, location, time, facebookUrl }) => (
-  <div className={s.card}>
-    <div className={s.cardHeader}>
-      <h1 className={s.cardDate}>
-        <b>{month} </b>
-        {date}
-      </h1>
-      <h2 className={s.cardDay}>{day}</h2>
-    </div>
-    <div className={s.cardBody}>
-      <h3 className={`${s.eventTitle} ${s[org.toLowerCase()]}`}>{title}</h3>
-      <h3 className={s.eventLocation}>{location}</h3>
-      <h3 className={s.eventTime}>{time}</h3>
-    </div>
-    <div className={s.cardFooter}>
-      <a role="button">
-        <img
-          src="/assets/calendar.svg"
-          alt=""
-          width={20}
-          height={20}
-          className={s.footerIcon}
-        ></img>
-      </a>
-      {!facebookUrl ? null : (
-        <a href={facebookUrl}>
-          <img src="/assets/facebook.svg" alt="" width={20} height={20} className={s.footerIcon} />
+  uuid: string;
+}> = ({ month, date, day, title, org, location, time, facebookUrl, uuid }) => (
+  <Link href={`/events/${formatURLEventTitle(title)}-${uuid}`} passHref>
+    <div className={s.card}>
+      <div className={s.cardHeader}>
+        <h1 className={s.cardDate}>
+          <b>{month} </b>
+          {date}
+        </h1>
+        <h2 className={s.cardDay}>{day}</h2>
+      </div>
+      <div className={s.cardBody}>
+        <h3 className={`${s.eventTitle} ${s[org.toLowerCase()]}`}>{title}</h3>
+        <h3 className={s.eventLocation}>{location}</h3>
+        <h3 className={s.eventTime}>{time}</h3>
+      </div>
+      <div className={s.cardFooter}>
+        <a role="button">
+          <img src="/assets/calendar.svg" alt="" width={20} height={20} className={s.footerIcon} />
```

Comment:
out of curiosity, why don't we use the next image component?

Reply:
next images are rly hard to style but i can definitely change it here since it's a fixed size

Reply:
specify this in a comment so it makes sense for future maintenance; that's an interesting but i guess logical choice by the library